pos_event_sale_session/static/src/js/models/EventTicket.js

```diff
@@ -0,0 +1,37 @@
+/*
+    Copyright 2022 Moka Tourisme (https://www.mokatourisme.fr).
+    @author Iván Todorovich <ivan.todorovich@gmail.com>
+    License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+*/
+odoo.define("pos_event_sale_session.EventTicket", function (require) {
+    "use strict";
+
+    const models = require("pos_event_sale.EventTicket");
+    const EventTicketSuper = models.EventTicket.prototype;
+
+    models.EventTicket = models.EventTicket.extend({
+        /**
+         * @override
+         * @param {Order} options.session Limit count to the given EventSession
+         * @returns {Number} ordered quantity
+         */
+        getOrderedQuantity: function () {
+            /* eslint-disable no-param-reassign */
+            // TODO: Disabled for now. Re-enable when we implement session ticket limits
```

Comment:
Maybe we can remove this file or implement it ? or if tickets limits require extra dependency do it in an other module ?!

Reply:
Indeed. I removed it because this should rather be implemented in `event_sale_session` first, and it's not trivial